ydb/core/mind/hive/monitoring.cpp

```diff
@@ -3298,6 +3310,62 @@ class TTxMonEvent_StopTablet : public TTransactionBase<THive> {
     void Complete(const TActorContext&) override {}
 };
 
+class TTxMonEvent_StopDomain : public TTransactionBase<THive>, TLoggedMonTransaction {
+public:
+    THolder<NMon::TEvRemoteHttpInfo> Event;
+    const TActorId Source;
+    TSubDomainKey DomainId;
+    bool Stop = true;
+
+    TTxMonEvent_StopDomain(const TActorId& source, NMon::TEvRemoteHttpInfo::TPtr& ev, TSelf* hive)
+        : TBase(hive)
+        , TLoggedMonTransaction(ev, hive)
+        , Event(ev->Release())
+        , Source(source)
+    {
+        ui64 ssId = FromStringWithDefault<ui64>(Event->Cgi().Get("ss"), 0);
+        ui64 pathId = FromStringWithDefault<ui64>(Event->Cgi().Get("path"), 0);
```

Comment:
It will be very difficult to realize that `path` is not a path (as a string) but an integer. `pathid` would be more appropriate.